components/dash-core-components/src/fragments/Graph.react.js

```diff
@@ -159,8 +159,10 @@ class PlotlyGraph extends Component {
             return Plotly.animate(gd, figure, animation_options);
         }
 
-        const configClone = this.getConfig(config, responsive);
         const layoutClone = this.getLayout(figure.layout, responsive);
+        const configClone = this.getConfig(config, responsive);
+        // add typesetMath | not exposed to the dash API
+        configClone.typesetMath = this.props.mathjax;
```

Comment:
Nonblocking, but ideally this would be baked into `this.getConfig(config, responsive, mathjax)`